tests/Find-DbaDbUnusedIndex.Tests.ps1

```diff
@@ -1,19 +1,136 @@
-$CommandName = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
+$script:UnusedIndexCommandName = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
```

Comment:
Please revert this change.

Reply:
Ok, I can revert that change. For some reason on my machine the scope of $CommandName causes the Get-Command on line 8 to fail because $CommandName is null in that block. When I had checked the Pester documentation it seemed to suggest that using the script scope was needed.

Reply:
Hmm. Was that referenced in our documentation?

General process I use that tends to with is importing the module psd1 and pester into the session. Then invoke the tests script.

Let me know if that was not understood or documented wrong and I'll get that updated.

Reply:
Thanks, that was the tip I needed. It seems running the test using Visual Studio Code was the issue with the variable scope since it starts the execution from a specific line number. I'm able to move forward now and I'm looking into an AppVeyor error that occurred in part 1 of the default scenario. 

Is there a process for running our own AppVeyor before creating a PR? I can help edit the instructional text and test it.

Reply:
A commit will trigger another test run on appveyor. You can format your commit as `(do find-dbadbunusedindex)` and that will trigger only the test for that function to run. Saves a bit of time until I can with out optimizing our test framework again.

Running the test locally though in your own environment our in a container should suffice in most situations.